packages/feedback/src/screenshot/components/imageEditorWrapper.css.ts

```diff
@@ -0,0 +1,154 @@
+import { DOCUMENT } from '../../constants';
+
+/**
+ * Creates <style> element for widget dialog
+ */
+export function createScreenshotAnnotateStyles(): HTMLStyleElement {
+  const style = DOCUMENT.createElement('style');
+
+  const surface200 = '#FAF9FB';
+  const gray100 = '#F0ECF3';
+
+  style.textContent = `
+  .canvas {
```

Comment:
I'd be careful with these class names as they can conflict with other css we are injecting into the shadow dom. I tried to use [BEM](https://getbem.com/) naming convention for the other components

Reply:
thanks for bringing that up, we would need to modify this for screenshots & cropping too!